src/transformer-components/transformerList.ts

```diff
@@ -627,6 +628,28 @@ const transformerList: TransformerList = {
       },
     },
   },
+  "Mutable Clone": {
```

Comment:
I wonder if this is the best name for this transformer--it's not clear to me why "clone" vs "copy" here, though I may just not be familiar with the established distinction between the two. That said, middle schoolers may not be either.

It might be nice if we chose something that conveys even stronger the fact that updates don't flow--I like "snapshot" for this reason but I understand it doesn't convey the mutability aspect very well. I'm not sure what would be best here. Thoughts? @stardust66 @P-bibs